plugins/CoreHome/lang/en.json

```diff
@@ -11,6 +11,7 @@
         "CloseWidgetDirections": "You can close this widget by clicking on the 'X' icon at the top of the widget.",
         "ChooseX": "Choose %1$s",
         "DataForThisReportHasBeenPurged": "The data for this report is more than %s months old and has been purged.",
+        "DataForThisReportHasBeenDisabled": "The archive for this plugin is disabled, please check %1$shttps:\/\/matomo.org\/faq\/how-to-disable-archiving-the-segment-reports-for-specific-plugins%2$s for more info",
```

Comment:
We should try to write messages that are useful for all kind of users. This one might be a bit too technical and the FAQ link might only be relevant for the admin. Guess something simple like `Segmentation is currently disabled for this report` would do it. Not sure if we maybe should mention to contact the admin for details 🤔 Maybe @tsteur has an even better suggestion for that.

And a general thing: We should not include an URL in a translation, as that might be error prone for translations. In that case you could have only used one placeholder to include everything, or link a text like `this FAQ` instead.